crates/bevy_remote/Cargo.toml

```diff
@@ -27,6 +27,10 @@ hyper = { version = "1", features = ["server", "http1"] }
 serde = { version = "1", features = ["derive"] }
 serde_json = { version = "1" }
 http-body-util = "0.1"
+tungstenite = "0.24"
+hyper-tungstenite = "0.15"
+futures-util = "0.3"
```

Comment:
I cannot find any existing bevy dependency that has the [send](https://docs.rs/futures-util/latest/futures_util/sink/trait.SinkExt.html#method.send) method so I have to pull `futures-util` in.